### PR TITLE
Added components for Alerts and Buttons

### PR DESCRIPTION
**Pull request by Anurag021**
I have added components for Alert and Buttons.
added files alert.css and button.css.
also added alert and button html code in docs.html.
image
image

### CODE REVIEW

File: Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    --success-alert-color: rgb(138, 185, 138);
+    --success-alert-text-color: rgb(4, 105, 4);
+    --warning-alert-color: rgb(255, 196, 147);
+    --warning-alert-text-color: rgb(250, 114, 3);
+    --alert-border-color: black;
+  }
+  
+  .components-alert-display{
+
+    border: 0.1rem solid var(--alert-border-color);
```
Comment (partha8) [inline Components/alert.css]: Vikram mentioned that you can use px when you want to set border size.

**Review by partha8 — COMMENTED** [review]
1. Are you using different colors for different components? If so, use only a set of colors, two to four, and have different tints and shades of those colors. Have one primary color which will show the theme of your site. You can refer to other team members' projects. Store these variables in a different CSS file. Your way of using colors is complicated and might result in inconsistency in your design.

2. You can avoid using rems in borders.

3. Rest it looks good

4. I also noticed that you are merging this branch into the main branch. You did the same for two other branches. You might want to look into that as you are required to merge only in the dev branch.

File: Components/button.css [hunk 2]
```diff
@@ -1,140 +1,136 @@
...
-    border: 0.1rem solid black;
-    width: 60%;
-  }
-  
-  .btn{
+.components-button-display{
+  display: flex;
+  flex-direction: row;
+  padding: 1rem;
+  border: 0.1rem solid black;
```
Comment (VLeads) [inline Components/button.css]: For border and border radius use px instead of rem.

**Review by VLeads — COMMENTED** [review]
Rest code looks good.

**Comment by VLeads** [comment]
Also I noticed that you are merging into main. It was asked to merge into dev branch.

**Comment by Anurag021** [comment]
Thanks @partha8  for doing detailed review of the code.. it really helps.
I have implemented all the mentioned changes.
for color part I have added all the color codes in main.css.

**Comment by Anurag021** [comment]
Thanks @VLeads for the review, it helps. 
I have implemented all mentioned changes and also changed the merging branch from main to dev.

**Review by partha8 — APPROVED** [review]

**Pull request merged**